fluentcheck/assertions_is/base_is.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any
+
+
+class IsBase:
+    __check = None
+
+    def __init__(self, object_under_test: Any):
+        self.object = object_under_test
+
+    def __call__(self, *args, **kwargs):
+        return self
+
+    @property
+    def check(self):
+        if not IsBase.__check:
+            from fluentcheck.classes import Check
```

Comment:
This part is pretty bad Python. But is necessary if you require `Check` and `Is` be defined in a single class.